### PR TITLE
Fix LoadBalance not check segment exists

### PR DESCRIPTION
**Pull request by yah01**
Signed-off-by: yah01 <yang.cen@zilliz.com>
/kind bug
related #19441

### CODE REVIEW

File: tests/python_client/testcases/test_utility.py [hunk 1]
```diff
@@ -1549,7 +1549,7 @@ def test_load_balance_with_src_node_not_exist(self):
         # load balance
         self.utility_wrap.load_balance(collection_w.name, invalid_src_node_id, dst_node_ids, sealed_segment_ids,
                                        check_task=CheckTasks.err_res,
-                                       check_items={ct.err_code: 1, ct.err_msg: "is not exist to balance"})
+                                       check_items={ct.err_code: 1, ct.err_msg: "not found in source node"})
```
Comment (zhuwenxing) [inline tests/python_client/testcases/test_utility.py:1552]: This test case is about invalid_src_node_id not invalid segment id, so the error message should not be changed

Comment (yah01) [reply tests/python_client/testcases/test_utility.py]: I found that this error message changed as well, just modified it again

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**